ironfish/src/rpc/clients/tcpClient.test.ts

```diff
@@ -23,6 +23,7 @@ describe('IronfishTcpClient', () => {
         data: {
           mid: messageId,
           type: route,
+          auth: 'test token',
```

Comment:
we don't have a test to test that this works when auth is undefined, which I think is fine. What do you think?

Reply:
added